src/lib/config.ts

```diff
@@ -73,7 +73,7 @@ const configSchema = z
         z.literal('localStorage'),
       ])
       .optional(),
-    tree: z.string(),
+    tree: z.string().optional(),
```

Comment:
I discovered that we need to discourage the setting of the `tree` value on the base config. It's best if they pass in a tree at actual call time.

Reply:
is there further reasoning as to why we should discourage this? All of our samples and docs kinda follow this pattern.

Reply:
I'll write something up in the Widget's docs about this today. That way, we will have it documented for ourselves and others.